src/Contracts/AnalysisTextControler.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Whatafix\TextTagger\Contracts;
+
+use Whatafix\TextTagger\Contracts\AnalysisTextLibrary;
+
+//création du contrôleur pour appeler la librairie
+class AnalysisTextControler
+{
+    //cette fonction est pour récupérer la fonction de la librairie 
+
+    public function split(string $text){
+
+     $text = new AnalysisTextLibrary();
+     $text->getTags('');
+     return $text;
+    }
+}
```

Comment:
Attention à la syntaxe.

```suggestion
/**
 * Une description de classe ce met en entête
 * Cette fonction est pour récupérer la fonction de la librairie 
 */
class AnalysisTextControler
{
    public function split(string $text) // Typage de retour ?
    { // retour à la ligne
         $text = new AnalysisTextLibrary();
         $text->getTags(''); // A quoi ça sert ? 
         // saut de ligne avant un return
         return $text;
    }
}
```